simple_httpfs/__main__.py

```diff
@@ -46,6 +46,12 @@ def main():
         '--aws-profile',
         default=None,
         type=str)
+    
+    parser.add_argument(
+        '-a', '--allow-other',
```

Comment:
Since no other argument includes a shortcut I'd suggest limit this argument to `--allow-other`
```suggestion
        '--allow-other',
```